src/containers/Menu.js

```diff
@@ -30,9 +32,10 @@ class Menu extends Component {
   }
 
   clickNavbarItem(event, item) {
+    const state = loadState()
```

Comment:
Use the session from the redux  store and not the localstorage. Check the home container for an example